src/Model.php

```diff
@@ -1789,18 +1789,12 @@ public function rawIterator(): iterable
     /**
      * Executes specified method or callback for each record in DataSet.
      *
-     * @param string|callable $method
-     *
      * @return $this
      */
-    public function each($method)
+    public function each(\Closure $method)
```

Comment:
`$method` -> `$fx`

Reply:
Of course, method is not optimal, so 👍 - I know, but should we use fx, func or fn? I past, I was using Func suffix a lot (like `$sortFunc`), but because of php uses `fn` shortcut for function, shouldň we use fn? Of course everywhere...

Reply:
done